src/build.cc

```diff
@@ -1033,14 +1033,15 @@ bool Builder::FinishCommand(CommandRunner::Result* result, string* err) {
   }
 
   if (!deps_type.empty() && !config_.dry_run) {
-    assert(edge->outputs_.size() == 1 && "should have been rejected by parser");
```

Comment:
Should we still assert that there is at least one output?

Reply:
I've added back the assert. I can't test it because `build: foo` is invalid because at least one output is required in the grammar.

Reply:
Rather, I can't test that the appropriate check in `manifest_parser` would be triggered; the error happens before that point).